src/common.js

```diff
@@ -478,6 +489,12 @@ export class ParameterInput extends React.Component {
       desc: "read_bytes_lines",
       tooltip: "Read up to this number of bytes or lines",
     },
+    reference_level: {
+      type: "float",
+      desc: "reference_level",
+      tooltip: "Volume level at which full low_boost/high_boost is applied." +
+        "<br>Boost is scaled down linearly to 0dB at reference_level + 20dB and above.",
```

Comment:
"Volume level at which low_boost/high_boost is starting to be applied. Boost is scaled up linearly to reach the full value at reference_level - 20dB. Above reference_level only gain is applied."